subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/ClasspathSnapshotter.java

```diff
@@ -19,7 +19,7 @@
 import org.gradle.internal.fingerprint.FileCollectionFingerprint;
 
 /**
- * Responsible for calculating a {@link FileCollectionFingerprint} for a {@link org.gradle.api.file.FileCollection} representing a Java classpath. Compared to {@link org.gradle.internal.fingerprint.RelativePathInputNormalizer} this fingerprinter orders files within any sub-tree.
+ * Responsible for calculating a {@link FileCollectionFingerprint} for a {@link org.gradle.api.file.FileCollection} representing a Java classpath. Compared to {@link org.gradle.internal.fingerprint.impl.RelativePathFileCollectionFingerprinter} this fingerprinter orders files within any sub-tree.
```

Comment:
Shouldn't this be in `org.gradle.internal.fingerprint` and called `ClasspathFingerprinter`?